docusaurus/website/sidebars.js

```diff
@@ -17,6 +17,11 @@ const sidebars = {
   // tutorialSidebar: [{ type: 'autogenerated', dirName: '.' }],
 
   docs: {
+    Introduction: [
+      'introduction/backend-plugins',
+      'introduction/grafana-plugin-sdk-for-go',
+      'introduction/plugin-protocol',
+      'introduction/data-frames',],
```

Comment:
Not sure if this fails CI but this file should be formatted via prettier:

```suggestion
      'introduction/data-frames',
    ],
```